### PR TITLE
Changed text + Animations

### LINKED ISSUE #239: Update Rewards onboarding motion graphics
### Description
The motion graphics in the onboarding flow needs to be updated:

1. On the Rewards slide, update the motion graphic with a new one that says "SomeBlog.org" instead of "Dailyblog.com" (we purchased the domain of the former and set up a redirect):
https://lottiefiles.com/share/sH12Fp
https://assets10.lottiefiles.com/packages/lf20_sH12Fp.json

2. On the ads notification slide, update the motion graphic with a new one that has a different ad image (Brad didn't like the spray can): 
https://lottiefiles.com/share/hn8gqM
https://assets1.lottiefiles.com/packages/lf20_hn8gqM.json

### JSON files
[lottie-export.zip](https://github.com/brave/brave-rewards-ios/files/3704884/lottie-export.zip)

### PR DESCRIPTION
**Pull request by Brandon-T**
Changed text to "Show Me" instead of "Join" since we aren't joining anything.

Fixes: https://github.com/brave/brave-ios/pull/1556
Fixes: https://github.com/brave/brave-rewards-ios/issues/239

## Summary of Changes

This pull request fixes issue #<number>

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by kylehickinson — COMMENTED** [review]
This is **only** for the last flow (rewards enabled, ad region available now), correct?

File: Client/Frontend/Browser/Onboarding/onboarding-ads.json [hunk 1]
```diff
@@ -1 +1 @@
...:65,"s":[224.601,45.246,0],"to":[-8.825,-124.262,0],"ti":[20.314,73.727,0]},{"t":85.000003462121,"s":[102.715,-397.117,0]}],"ix":2},"a":{"a":0,"k":[0,0,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":0,"ef":[{"ty":25,"nm":"Drop Shadow","np":8,"mn":"ADBE Drop Shadow","ix":1,"en":1,"ef":[{"ty":2,"nm":"Shadow Color","mn":"ADBE Drop Shadow-0001","ix":1,"v":{"a":0,"k":[0.152941003442,0.180391997099,0.250979989767,0.20000000298],"ix":1}},{"ty":0,"nm":"Opacity","mn":"ADBE Drop Shadow-0002","ix":2,"v":{"a":0,"k":51,"ix":2}},{"ty":0,"nm":"Direction","mn":"ADBE Drop Shadow-0003","ix":3,"v":{"a":0,"k":180,"ix":3}},{"ty":0,"nm":"Distance","mn":"ADBE Drop Shadow-0004","ix":4,"v":{"a":0,"k":15,"ix":4}},{"ty":0,"nm":"Softness","mn":"ADBE Drop Shadow-0005","ix":5,"v":{"a":0,"k":33,"ix":5}},{"ty":7,"nm":"Shadow Only","mn":"ADBE Drop Shadow-0006","ix":6,"v":{"a":0,"k":0,"ix":6}}]}],"shapes":[{"ty":"gr","it":[{"d":1,"ty":"el","s":{"a":0,"k":[69,67.444],"ix":2},"p":{"a":0,"k":[0,0],"ix":3},"nm":"Ellipse Path 1","mn":"ADBE Vector Shape - Ellipse","hd":false},{"ty":"fl","c":{"a":0,"k":[0.517647027969,0.368627011776,0.968626976013,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn
```
Comment (kylehickinson) [inline Client/Frontend/Browser/Onboarding/onboarding-ads.json:1]: ?

File: Client/Frontend/Browser/Onboarding/onboarding-rewards.json [hunk 2]
```diff
@@ -1 +1 @@
...6,0],[-0.43,-0.27],[0,0],[0,0],[0,0],[0,0],[0,0],[0.2,-0.46],[0.43,-0.27],[0.65,0]],"v":[[9.441,-3.656],[10.652,-2.266],[11.092,-0.137],[10.652,1.993],[9.441,3.383],[7.712,3.874],[6.171,3.464],[5.232,2.363],[5.232,7.493],[2.842,7.493],[2.842,-4.036],[5.232,-4.036],[5.232,-2.626],[6.171,-3.727],[7.712,-4.137]],"c":true},"ix":2},"nm":"Path 2","mn":"ADBE Vector Shape - Group","hd":false},{"ind":2,"ty":"sh","ix":3,"ks":{"a":0,"k":{"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[-1.764,3.771],[-1.764,-4.038],[0.626,-4.038],[0.626,3.771]],"c":true},"ix":2},"nm":"Path 3","mn":"ADBE Vector Shape - Group","hd":false},{"ind":3,"ty":"sh","ix":4,"ks":{"a":0,"k":{"i":[[-0.26,0.23],[-0.43,0],[-0.26,-0.24],[0,-0.36],[0.26,-0.24],[0.43,0],[0.26,0.24],[0,0.36]],"o":[[0.26,-0.24],[0.43,0],[0.26,0.23],[0,0.36],[-0.26,0.24],[-0.43,0],[-0.26,-0.24],[0,-0.36]],"v":[[-1.598,-7.137],[-0.559,-7.497],[0.471,-7.137],[0.872,-6.236],[0.471,-5.346],[-0.559,-4.986],[-1.598,-5.346],[-1.988,-6.236]],"c":true},"ix":2},"nm":"Path 4","mn":"ADBE Vector Shape - Group","hd":false},{"ind":4,"ty":"sh","ix":5,"ks":{"a":0,"k":{"i":[[0,0],[0,0],[0,0],[0,0],[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,
```
Comment (kylehickinson) [inline Client/Frontend/Browser/Onboarding/onboarding-rewards.json:1]: ?

**Review by kylehickinson — CHANGES_REQUESTED** [review]
Can we dump those 2 random JSON file changes, other then that, change is good

File: Client/Frontend/Browser/Onboarding/onboarding-ads.json (hunk 1 above)
Comment (Brandon-T) [reply Client/Frontend/Browser/Onboarding/onboarding-ads.json:1]: This is part of the ticket. https://github.com/brave/brave-rewards-ios/issues/239
I was messaged to add it in last night on Slack so I added it but only pushed it this morning.
Basically the animation changed for both the REWARDS and ADS screen.

File: Client/Frontend/Browser/Onboarding/onboarding-rewards.json (hunk 2 above)
Comment (Brandon-T) [reply Client/Frontend/Browser/Onboarding/onboarding-rewards.json:1]: This is part of the ticket. https://github.com/brave/brave-rewards-ios/issues/239

**Review by kylehickinson — APPROVED** [review]

**Pull request merged**